### ISSUE TITLE
Modelling of heterogeneity

### ISSUE THREAD

**Issue by jgroehl** [IMSY-DKFZ/simpa#154]
Currently, the tissue models are homogeneous, both from the segmentation-based loader, as well as the model-based loader. We've shown in a prior publication [CE-qPAI](https://www.spiedigitallibrary.org/journals/journal-of-biomedical-optics/volume-23/issue-5/056008/Context-encoding-enables-machine-learning-based-quantitative-photoacoustics/10.1117/1.JBO.23.5.056008.full?SSO=1) that modelling heterogeneity can make a difference e.g. in terms of generalizability of trained models and also for the distribution of initial pressure values in the simulated data.

As we'll be needing this feature for an upcoming project I'll be working on this.

Have you already implemented anything in this regard that has not made it to the code base yet and could be shared @kdreher @mschllnbrg?

**Comment by kdreher** [comment]
No, I haven't worked on this any further except for thinking a lot about this, and I think I have some ideas of how to implement this nicely. 
I would be open to brainstorming about this if you have time :)

**Comment by mschllnbrg** [comment]
@kdreher @jgroehl 

We could also try to include natural images, e.g. images of grassy landscapes, to add some heterogeneous structure. Just an idea but it might be worth a try.
